frontend/packages/topology/src/utils/application-utils.ts

```diff
@@ -237,6 +239,10 @@ export const cleanUpWorkload = async (
     case KnativeServiceModel.kind:
       batchDeleteRequests(knativeDeleteModels, resourceData);
       break;
+    case CamelKameletBindingModel.kind:
+    case KafkaSinkModel.kind:
+      deleteRequest(resourceModel, resource);
+      break;
```

Comment:
Can you add test for this application-utils.

Reply:
done